style.css

```diff
@@ -0,0 +1,996 @@
+  }
+
+  .index {
+    position: relative;
+    z-index: 2;
+  }
+
+  .before-nav {
+    width: 100%;
+    background-color: #272a31;
+    margin-top: 0;
+  }
+
+  .before-nav-content {
+    height: 40px;
+    color: #d3d3d3;
+    display: flex;
+    justify-content: flex-end;
+    padding: 0 10%;
+    gap: 20px;
+    align-items: center;
+  }
+
+  .before-nav-content ul {
+    text-decoration: none;
+  }
+
+  nav {
+    display: flex;
+    justify-content: space-around;
+    background-color: #fff;
+    margin-top: -1px;
+    height: 35px;
+  }
+
+  .logo a {
+    font-size: 1.5rem;
+    color: rgb(0, 0, 0);
+    text-decoration: none;
+  }
+
+  .nav-links {
+    display: flex;
+  }
+
+  .desktop-navlinks {
+    display: flex;
+    gap: 60px;
+    align-items: center;
+  }
+
+  .nav-links li {
+    list-style: none;
+    font-family: "Lato", sans-serif;
+  }
+
+  .nav-links a {
+    text-decoration: none;
+    color: #272a31;
+  }
+
+  #closeMenu {
+    display: none;
+  }
+
+  .fa-solid {
+    font-family: "Font Awesome 6 Free", sans-serif;
+    font-weight: 900;
+  }
+
+  .nav-button {
+    border: #ec5242 2px solid;
+    background-color: #fff;
+    color: #ec5242;
+    padding: 5px;
+    font-size: 18px;
+  }
+
+  .nav-button:hover {
+    cursor: pointer;
+    background-color: #ec5242;
+    color: #fff;
+  }
+
+  .introduction {
+    margin: 8%;
+  }
+
+  .first-heading {
+    color: #ec5242;
+  }
+
+  .second-heading {
+    margin-top: 3px;
+    margin-bottom: 10px;
+    font-family: sans-serif;
+    font-weight: 900;
+    line-height: 1.1em;
+    background-image: url(./images/textBack.jpg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: 10%;
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent;
+  }
+
+  .para-border,
+  .para-border-about {
+    padding: 20px 12px;
+    margin-top: 20px;
+    margin-bottom: 25px;
+    border: 2px solid rgba(243, 241, 241, 0.6);
+    background-color: rgba(255, 255, 255, 0.2);
+    backdrop-filter: blur(60px);
+    -webkit-backdrop-filter: blur(60px);
+    line-height: 1.3em;
+    letter-spacing: 1.5px;
+  }
+
+  .venue h2 {
+    font-weight: bold;
+    font-size: 35px;
+  }
+
+  .burger {
+    display: none;
+  }
+
+  .line {
+    display: none;
+  }
+
+  .main {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    background-color: #272a31;
+    color: #fff;
+    background-image: url(./images/backPattern.png);
+    margin-top: -2px;
+    padding: 50px 10px;
+  }
+
+  hr {
+    background-color: #ec5242;
+    height: 3px;
+    width: 80px;
+    border: none;
+    margin: 27px auto;
+  }
+
+  .main-grids {
+    display: flex;
+    gap: 10px;
+  }
+
+  .box {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    background-color: rgba(255, 255, 255, 0.1);
+    border-radius: 5px;
+    max-width: 200px;
+    height: 15%;
+    border: none;
+    padding: 20px;
+    margin: 2px;
+  }
```

Comment:
- [ ] Kindly make sure to add the missing `border` effect when the user hover any box in the program (main-grids section} as shown in the image below 👇🏻 :

![main](https://user-images.githubusercontent.com/79058364/169443599-e58401a3-358c-4dab-ae4f-b51f3638bbf9.png)